src/app/static/src/js/components/runReport/changeLog.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <div v-if="showChangeMessage">
```

Comment:
If we retain this (see below) then would make more sense for it to be `showChangelog` rather than `showChangeMessage`, as it covers both `message` and `type`